hoomd/md/EvaluatorExternalElectricField.h

```diff
@@ -41,9 +41,35 @@ class EvaluatorExternalElectricField
     {
     public:
     //! type of parameters this external potential accepts
-    typedef struct param
+    struct param_type
         {
-        } param_type;
+        Scalar3 E;
+
+#ifndef __HIPCC__
+        param_type() : E(make_scalar3(0, 0, 0)) { }
+
+        param_type(pybind11::object params)
+            {
+            pybind11::tuple py_E(params);
+            E.x = pybind11::cast<Scalar>(py_E[0]);
+            E.y = pybind11::cast<Scalar>(py_E[1]);
+            E.z = pybind11::cast<Scalar>(py_E[2]);
+            }
+
+        pybind11::object asDict()
```

Comment:
This can be done for all external potentials (i.e. `PotentialExternalElectricField` and `PotentialExternalPeriodic`), converting all evaluators throughout the codebase might be out of scope for this PR.